From 5f64275aa1ddcf37c87599d1fae6dab8d58bb976 Mon Sep 17 00:00:00 2001
From: Adam Jackson <ajax@redhat.com>
Date: Thu, 7 Sep 2017 17:02:22 -0400
Subject: [PATCH 5/5] dispatch: Fix the libOpenGL soname

Brown-paper-bag-for: Adam Jackson <ajax@redhat.com>
(cherry picked from commit e5372a25baa9034b6223b32a0cab838c42779a39)
---
 src/dispatch_common.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/dispatch_common.c b/src/dispatch_common.c
index 87bd98a..7ec91b6 100644
--- a/src/dispatch_common.c
+++ b/src/dispatch_common.c
@@ -194,7 +194,7 @@
 #define EGL_LIB "libEGL.so.1"
 #define GLES1_LIB "libGLESv1_CM.so.1"
 #define GLES2_LIB "libGLESv2.so.2"
-#define OPENGL_LIB "libOpenGL.so.1"
+#define OPENGL_LIB "libOpenGL.so.0"
 #endif
 
 #ifdef __GNUC__
-- 
2.13.5

